pkg/chartrenderer/testdata/alpine/templates/alpine-charts.yaml

```diff
@@ -22,3 +22,17 @@ metadata:
     chartName: {{ .Chart.Name }}
     chartVersion: {{ .Chart.Version | quote }}
 type: Opaque
+---
```

Comment:
Please rename the file to something more meaningful - `alpine-resources.yaml` or `alpine.yaml`.